modules/mono/editor/bindings_generator.cpp

```diff
@@ -1210,92 +1210,88 @@ Error BindingsGenerator::_generate_cs_type(const TypeInterface &itype, const Str
 
 	output.append(INDENT1 "{");
 
-	if (class_doc) {
-		// Add constants
+	// Add constants
 
-		for (const List<ConstantInterface>::Element *E = itype.constants.front(); E; E = E->next()) {
-			const ConstantInterface &iconstant = E->get();
-
-			if (iconstant.const_doc && iconstant.const_doc->description.size()) {
-				String xml_summary = bbcode_to_xml(fix_doc_description(iconstant.const_doc->description), &itype);
-				Vector<String> summary_lines = xml_summary.length() ? xml_summary.split("\n") : Vector<String>();
+	for (const List<ConstantInterface>::Element *E = itype.constants.front(); E; E = E->next()) {
+		const ConstantInterface &iconstant = E->get();
 
-				if (summary_lines.size()) {
-					output.append(MEMBER_BEGIN "/// <summary>\n");
+		if (iconstant.const_doc && iconstant.const_doc->description.size()) {
+			String xml_summary = bbcode_to_xml(fix_doc_description(iconstant.const_doc->description), &itype);
+			Vector<String> summary_lines = xml_summary.length() ? xml_summary.split("\n") : Vector<String>();
 
-					for (int i = 0; i < summary_lines.size(); i++) {
-						output.append(INDENT2 "/// ");
-						output.append(summary_lines[i]);
-						output.append("\n");
-					}
+			if (summary_lines.size()) {
+				output.append(MEMBER_BEGIN "/// <summary>\n");
 
-					output.append(INDENT2 "/// </summary>");
+				for (int i = 0; i < summary_lines.size(); i++) {
+					output.append(INDENT2 "/// ");
+					output.append(summary_lines[i]);
+					output.append("\n");
 				}
-			}
 
-			output.append(MEMBER_BEGIN "public const int ");
-			output.append(iconstant.proxy_name);
-			output.append(" = ");
-			output.append(itos(iconstant.value));
-			output.append(";");
+				output.append(INDENT2 "/// </summary>");
+			}
 		}
 
-		if (itype.constants.size())
-			output.append("\n");
+		output.append(MEMBER_BEGIN "public const int ");
+		output.append(iconstant.proxy_name);
+		output.append(" = ");
+		output.append(itos(iconstant.value));
+		output.append(";");
+	}
 
-		// Add enums
+	if (itype.constants.size())
```

Comment:
Please add braces around single-statement `if` and `else` blocks.
Running `clang-tidy --fix` on the file might help (I've only tried it with a compilation database and it takes ages with lots of broken results in thirdparty code, but it might work fine on a single file.)

Reply:
Sure. Would be great to have a commit hook for this.

Reply:
I'm not sure it's possible, `clang-tidy` is much harder to use than `clang-format`. It requires passing all the defines and includes like the buildsystem would, otherwise it will throw errors and fail applying the style changes. Alternatively, it can be used via a compilation database which we now generated, but it then applies to the whole codebase (including `thirdparty`) and takes ~20 min on my i7 -- often failing due to conflicting changes in `thirdparty`.